Directory.Build.props

```diff
@@ -16,6 +16,7 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PreviousVersion>net6.0</PreviousVersion>
+    <LegacyVersion>netstandard2.0</LegacyVersion>
```

Comment:
I think you can skip this build property and just include the new target framework value as-is